ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_15/InterfaceSubscriptionsIT.scala

```diff
@@ -481,8 +481,15 @@ class InterfaceSubscriptionsIT extends LedgerTestSuite {
       )
       assertEquals(
         "1 and 3 produce the same views (but not the same create arguments)",
-        transactions1.map(updateTransaction()),
-        transactions3.map(updateTransaction(emptyContractKey = true, emptyCreateArguments = true)),
+        // do not check on details since tid is contained and it is expected to be different
+        transactions1.map(updateTransaction(emptyDetails = true)),
```

Comment:
I don't understand why the traceId related changes blew up in this place. It seems unrelated to the subject.

Reply:
The details of the status of the interface views contains the tid:
```
DAML_INTERPRETATION_ERROR
participant'participant1')
tid"'9463676f15dff8cebe9b2f3599be8baa'
category9
definite_answerfalse
```
and so for different transactions the tids differ.

Reply:
On failure, the interface view grpc `Status` is constructed by means of the self-service error code (which inlines property map of the `LoggingContext`, which has the `traceId` now) - see https://github.com/digital-asset/daml/blob/main/ledger/participant-integration-api/src/main/scala/platform/store/dao/events/LfValueTranslation.scala#L437-L450

Reply:
I don't see why we should remove it, but I think your approach with ignoring the status details is fine 👍

Reply:
It is absolutely fine. It just exposes the fact that our current method of asserting on the error message is extremely brittle